composer.json

```diff
@@ -212,6 +214,9 @@
                 "Configure core to not check permissions before accepting an updated module": "patches/disable-core-upload-permission-check.diff",
                 "3073554: Add a token for publication status": "https://www.drupal.org/files/issues/2019-08-09/3073554_10.patch"
             },
+            "drupal/focal_point": {
+                "Preview link accidentally closes the media library\n": "https://www.drupal.org/files/issues/2020-10-11/preview_link_accidentally_closes_the_media_library-3162210-19.patch"
```

Comment:
Please include the node id for the patch in the title like we do for the other external patches.

Is it intentional that you include a newline `\n`?